content/community/conferences.md

```diff
@@ -22,6 +22,11 @@ April 1st, 2022. Amsterdam, The Netherlands
 
 [Website](https://www.reactlive.nl/) - [Twitter](https://twitter.com/reactlivenl)
 
+### < React Global > by Geekle {#react-global-2022}
+April 21st-22nd, 2022 - Online Summitt
```

Comment:
```suggestion
April 21st-22nd, 2022 - Online Summit
```

Reply:
yes, it is April 21st-22nd, 2022 - Online Summit

Reply:
Can we change it to `April 21-22, 2022 - Online Summit`. I see that older conferences were consistent and let's follow that?